composer.json

```
@@ -20,10 +20,10 @@
     },
     "require": {
         "ext-json": "*",
-        "php": ">=7.1"
+        "php": ">=7.1 || ^8.0"

```

```suggestion
        "php": "^7.1 || ^8.0"

```
`>=7.1` Should also allow 8.0.
however i like the more strictness of `^7.1 || ^8.0` because this will lockout 9.0